controllers/clustermesh/clustermesh_controller.go

```diff
@@ -437,9 +548,10 @@ func ReconcileSecurityGroups(r *ClusterMeshReconciler, ctx context.Context, clus
 	if err != nil {
 		return err
 	}
-	r.log.Info(fmt.Sprintf("successfully %s security group %s for cluster %s in clustermesh %s\n", string(operationResult), sg.Name, cluster.Name, clustermesh.Name))
+	r.log.Info(fmt.Sprintf("successfully %s security group %s for cluster %s in clustermesh %s\n", string(operationResult), sg.Name, cluster.Name, r.clustermesh.Name))
 
-	err = controllerutil.SetOwnerReference(clustermesh, sg, r.Scheme)
+	// TODO: por que náo é feito no createorupdate?
```

Comment:
opiniões aqui?

Reply:
I vaguely remember that we had some problems setting owner reference directly in the object and using the external lib function, but If you test it and this works I don't see a reason why not